NewEvaluation.html

```diff
@@ -654,20 +643,18 @@
 			</fieldset>
 		</form>
 
-	<div class="littleSpacing"></div>
-
-	<form  action="Evaluation.html">
-		<input  type="submit" name="button" value="Submit">
-	</form>
+		<div class= "submit-button">
+			<form action="Evaluation.html">
+				<input  type="submit" name="button" value="Submit">
```

Comment:
Yes. Well, sort of... You see, in HTML, when you submit a form, you only submit the data that is _part of that particular form_. Multiple forms? Fragmented data. So, then, what's happening in this specific case? The only data that gets sent to the "backend" (currently, `Evaluation.html`) is a parameter named `button` with a value of `Submit`. But what about the rest of the data that you've input into other forms? Well, tough luck... it's now gone. :)